test/regression.uts

```diff
@@ -641,6 +641,15 @@ DNS in dns_ans
 ~ netaccess IP
 IP(src="8.8.8.8").whois()
 
+= AS resolvers
+~ netaccess IP
+
+ret = conf.AS_resolver.resolve("8.8.8.8", "8.8.4.4")
+len(ret) == 2
```

Comment:
You might want to use `assert` here, since this is not the last line of the test.